### ISSUE TITLE
Add a SIP Gateway to Scalelite

### ISSUE THREAD

**Issue by pielonet** [blindsidenetworks/scalelite#256]
Hi,

Bigbluebutton offers the opportunity, thanks to Freeswitch, to set up a SIP bridge to access a conference by phone.

I couldn't find this functionnality in Scalelite.

Is there a possibility to setup a unique phone number and be redirected to the correct BigBlueButton server and conference by Scalelite when dialing the PIN number ?

Kind regards,
Thierry

**Comment by sam9032** [comment]
Any news on this topic?
It would be amazing, having this feature in BBB.
Thank you all for the excellent work! 😃

**Comment by hokascha** [comment]
> Doesn't that have to be configured in each BBB instance instead of scalelite?

Yes, at the moment, you're right. But it would be nice to have only one single dial in number and route traffic to the corresponding BBB instance. Not sure if this is even possible. I think that would require a FreeSWITCH instance at scalelite for connecting to the voip provider.

**Comment by mschwrdtnr** [comment]
Does anyone have ideas or more concrete suggestions for implementation and how this can be done?

**Comment by Franz-Heinz** [comment]
Notice: Until this is solved greenlight is putting on a hold a PR that would benefit a lot of users: https://github.com/bigbluebutton/greenlight/pull/2159
So it would be great if there was to be a solution for this.

**Comment by bwildenhain** [comment]
> Does anyone have ideas or more concrete suggestions for implementation and how this can be done?

I use those Asterisk together with https://github.com/MBcom/bbb-clustersip to redirect external calls to the correct node from my common dial-in number.
